06 - Type Ahead/style.css

```diff
@@ -72,3 +72,7 @@ span.population {
 .hl {
   background: #ffc600;
 }
+
+.errorMessage{
+  color: red;
+}
```

Comment:
always add a newline at the end of a file. I don't know why, but it is good practice.

Reply:
Done! Maybe to leave very long hangry comments from your reviewer on GitHub :)